raw_io.orogen

```diff
@@ -6,5 +6,20 @@ using_library "raw_io"
 import_types_from "raw_io/Analog.hpp"
 import_types_from "raw_io/Digital.hpp"
 
+import_types_from "raw_ioTypes.hpp"
+
 typekit.export_types "raw_io/Analog", "raw_io/Digital"
 typekit.export_types "/std/vector<raw_io/Analog>", "/std/vector<raw_io/Digital>"
+
+task_context "LinearTemperatureSensorTask" do
+  # This is the default from now on, and should not be removed. Rock will
+  # transition to a setup where all components use a configuration step.
+  needs_configuration
+
+  property "sensor_config", "/raw_io/TemperatureSensorSettings"
```

Comment:
LinearTemperatureSensorTask -> LinearTemperatureSensorSettings

'Linear' being the important keyword here.